Documentation/Conditions/Index.rst

```diff
@@ -67,13 +67,21 @@ page
    Array
 
 :aspect:`Description`
-   Current page record as array.
+   You can read all page information from the page array. To find out, what variable has which name, you can simply check the input fields in backend by inspecting the input name. Or you can enable BE Debug Mode, to see the name of the variables.
 
 :aspect:`Example`
    ::
 
       [page["uid"] == 2]
-
+      
+      # Or when you need the page backend layout in V9.4 or V10:
+      [page["backend_layout"] == 5]
+      
+      # Or the backend layout has a name:
+      [page["backend_layout"] == "example_layout"]
+      
+      #Maybe you want to check the page title 
+      [page["title"] == "foo"]
```

Comment:
TBH this example could be improved. The title is highly dependent on user input and can be changed any time. So this is not something code can rely on.

Reply:
Problem with these docs is, it is highly intransparent, where to find information, nor there is some information for it to find out, where to find the right use for it. But sure, the text can be minidied like this. im totally okay with that. I really just want to show (As a person who is new to TYPO3) how to easily access page variables.

Reply:
Technically you need to look at the schema of the `pages` table in your database since that's the definitive source. Even the backend form or TCA won't give you the full lists of fields.

Reply:
And thats what i mean, that info for the schema of "pages" should be linked in the docs as well. Is there a full list for that? I can update that as well. I think that would really help, finding out, what potential "page" array has.

Reply:
No, that's not possible since it depends on the extensions you have installed and activated. Any extension can adjust the schema of any table to add or even change fields.